controller/server/update_client.mli

```diff
@@ -0,0 +1,35 @@
+(* HTTP Client for gettig updates and their metadata from
+   the remote server. *)
+
+(* unparsed semver version *)
+type version = string
+(* local filesystem path *)
+type bundle_path = string
```

Comment:
Abstract types can be a bit heavy if all you need is e.g. a numeric value for representing seconds. But I think you do need either type aliases or named fun arguments for documentation purposes, otherwise you get interfaces with signatures like `download_bundle: string -> string -> string Lwt.t`, which can be very confusing.

I guess we can decide on a convention of named arguments rather than type aliases?